### DISCUSSION TITLE
Erlang version won't update

### DISCUSSION THREAD

**Discussion by oshybrid** [Homebrew/discussions#789] [Getting started]
I installed the latest version of erlang.

```
~ % brew upgrade erlang
Warning: erlang 23.2.4 already installed
```

But my running version remains to be just 23

```
% elixir -v
Erlang/OTP 23 [erts-11.1.7] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [hipe] [dtrace]

Elixir 1.11.3 (compiled with Erlang/OTP 23)
```

For some reason, it is still pointing to the standard version of erlang on my system

```
 ~ % which erl
/usr/local/bin/erl
```

Instead of the brew version.  I even pointed my profile to the version installed by homebrew under "cellar."

**Comment by gromgit** [comment]
Upstream chose not to print the full OTP version number in the `erl` header. Instead, they provide instructions for [retrieving such info when required](https://erlang.org/doc/system_principles/versions.html#retrieving-current-otp-version):
> In an installed OTP development system, the OTP version can be read from the text file `<OTP installation root>/releases/<OTP release number>/OTP_VERSION`. The absolute path to the file can by constructed by calling `filename:join([code:root_dir(), "releases", erlang:system_info(otp_release), "OTP_VERSION"])`.

So [one way of getting the full OTP release version](https://stackoverflow.com/a/34326368/4432299) is:
```bash
$ erl -eval '{ok, Version} = file:read_file(filename:join([code:root_dir(), "releases", erlang:system_info(otp_release), "OTP_VERSION"])), io:fwrite(Version), halt().' -noshell
23.2.4
```
